stubs/urllib3/urllib3/connectionpool.pyi

```diff
@@ -87,35 +96,33 @@ class HTTPConnectionPool(ConnectionPool, RequestMethods):
     ): ...
 
 class HTTPSConnectionPool(HTTPConnectionPool):
-    scheme: Any
-    ConnectionCls: Any
-    key_file: Any
-    cert_file: Any
-    cert_reqs: Any
-    ca_certs: Any
-    ssl_version: Any
-    assert_hostname: Any
-    assert_fingerprint: Any
+    key_file: str | None
+    cert_file: str | None
+    cert_reqs: int | str | None
+    ca_certs: str | None
+    ssl_version: int | str | None
+    assert_hostname: str | bool | None
```

Comment:
It's `str | Literal[False] | None` on the `main` branch https://github.com/urllib3/urllib3/blob/1e985050bb498f15895478bd61acc002be83ae65/src/urllib3/connectionpool.py#L942

```suggestion
    assert_hostname: str | Literal[False] | None
```

Reply:
Thanks, this was my misunderstanding on the use of Literal again — appreciate the find!